skimage/feature/tests/test_peak.py

```diff
@@ -272,14 +312,14 @@ def test_adjacent_different_objects():
 
 
 def test_four_quadrants():
-    image = np.random.uniform(size=(40, 60))
-    i, j = np.mgrid[0:40, 0:60]
-    labels = 1 + (i >= 20) + (j >= 30) * 2
+    image = np.random.uniform(size=(20, 30))
```

Comment:
It is not a small task for a larger lib, see: https://github.com/matplotlib/matplotlib/pull/5405